client/src/index.js

```diff
@@ -20,6 +20,7 @@ import { UserCoordinator } from "./user";
 import { Sentry } from "./utils/helpers/sentry";
 import { Url, createCoreApiVersionedUrlConfig } from "./utils/helpers/url";
 import { AppErrorBoundary } from "./error-boundary/ErrorBoundary";
+import { validateAppParams } from "./utils/context/appParams.utils";
```

Comment:
(See comment in appParams.utils)
```suggestion
import { validatedAppParams } from "./utils/context/appParams.utils";
```